docker/from-source/Debian

```diff
@@ -2,19 +2,23 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
-FROM debian:buster
+FROM debian:stable
 
 # get the latest fixes
 # install LibreOffice run-time dependencies
 # install adduser, findutils, openssl and cpio that we need later
 # install an editor
 # tdf#117557 - Add CJK Fonts to Collabora Online Docker Image
+RUN apt-get clean && apt-get update
+
+
 RUN apt-get update && \
     apt-get -y install libpng16-16 fontconfig adduser cpio \
                findutils nano libcap2-bin openssl inotify-tools \
-               procps libubsan0 libubsan1 openssh-client fonts-wqy-zenhei \
+               procps libubsan1 openssh-client fonts-wqy-zenhei \
                fonts-wqy-microhei fonts-droid-fallback fonts-noto-cjk
 
+#libubsan0 unable to locate package
```

Comment:
Probably this comment is unnecessary, if libubsan0 is not there in stable Debian.

Reply:
Okay will remove it